javascript/example/webgl_loader_draco.html

```diff
@@ -41,8 +41,8 @@
         </div>
         <pre id="fileDisplayArea"><pre>
     </div>
-    <script src="three.js"></script>
-    <script src="draco_decoder.js"></script>
+    <script src="https://cdn.rawgit.com/mrdoob/three.js/r84/build/three.min.js"></script>
+    <script src="../draco_decoder.js"></script>
```

Comment:
We discussed it and it should be ok. It may be better to make it more explicit that the server needs to be run from either the root location or the javascript/ folder (update the new step (2)).